### PR TITLE
[Bulk] Validation: validation fixes JM 5/23

### PR DESCRIPTION
**Pull request by jmart1428**
This PR fixes 30 missing descriptions.

### CODE REVIEW

File: iis/extensions/smooth-streaming-client/ismoothstreamingcache-endpersist-method-microsoft-web-media-smoothstreaming_1.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
 title: ISmoothStreamingCache.EndPersist Method  (Microsoft.Web.Media.SmoothStreaming)
```
Comment (AqMoh) [inline iis/extensions/smooth-streaming-client/ismoothstreamingcache-endpersist-method-microsoft-web-media-smoothstreaming_1.md:2]: If you want, let's remove the extra space between Method and (Microsoft...) in the title

File: iis/extensions/smooth-streaming-client/streaminfo-methods-microsoft-web-media-smoothstreaming_1.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ---
 title: StreamInfo Methods (Microsoft.Web.Media.SmoothStreaming)
 TOCTitle: StreamInfo Methods
+description: "List of members exposed by the StreamInfo type including the method type, name, and description of each."
```
Comment (AqMoh) [inline iis/extensions/smooth-streaming-client/streaminfo-methods-microsoft-web-media-smoothstreaming_1.md:4]: "...type, including..."

File: iis/extensions/smooth-streaming-client/streamupdatedeventargs-constructor-microsoft-web-media-smoothstreaming.md [hunk 3]
```diff
@@ -1,6 +1,7 @@
 ---
 title: StreamUpdatedEventArgs Constructor  (Microsoft.Web.Media.SmoothStreaming)
```
Comment (AqMoh) [inline iis/extensions/smooth-streaming-client/streamupdatedeventargs-constructor-microsoft-web-media-smoothstreaming.md:2]: Extra space between Constructor and (Microsoft...)

File: iis/extensions/transform-manager/jobdefinition-properties-microsoft-web-media-transformmanager.md [hunk 4]
```diff
@@ -1,6 +1,7 @@
 ---
 title: JobDefinition Properties (Microsoft.Web.Media.TransformManager)
 TOCTitle: JobDefinition Properties
+description: "List of members exposed by the JobDefinition type including the property type, name, and description of each."
```
Comment (AqMoh) [inline iis/extensions/transform-manager/jobdefinition-properties-microsoft-web-media-transformmanager.md:4]: "...type, including..."

File: iis/extensions/transform-manager/scheduler-canceljob-method-microsoft-web-media-transformmanager.md [hunk 5]
```diff
@@ -1,6 +1,7 @@
 ---
 title: Scheduler.CancelJob Method  (Microsoft.Web.Media.TransformManager)
```
Comment (AqMoh) [inline iis/extensions/transform-manager/scheduler-canceljob-method-microsoft-web-media-transformmanager.md:2]: Extra space between Method and (Microsoft...)

**Comment by jmart1428** [comment]
**Global effort to fix build validation errors**
@Rick-Anderson @mcleblanc 
The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you'd like to see. Thank you!

**Review by Rick-Anderson — APPROVED** [review]

**Pull request merged**